src/options/options.html

```diff
@@ -19,35 +19,35 @@
 				<a href="#">
 					<button type="button" class="message-action-button micro-button info invisible"></button>
 				</a>
-				<img class="icon-dismiss invisible" src="/common/img/close.svg" width="24" height="24" tabindex="0" data-i18n data-i18n-aria-label="__MSG_dismissIconDescription__"></span>
+				<img class="icon-dismiss invisible" src="/common/img/close.svg" width="24" height="24" tabindex="0" data-i18n data-i18n-aria-label="__MSG_dismissIconDescription__">
```

Comment:
FYI wtf was there this crazy span attribute? Indeed seems like a bug. :D Thanks!

Reply:
No problem. I ran the options page for Unicodify through an HTML validator while working on https://github.com/rugk/unicodify/pull/72 and it caught this issue, which I noticed was also present here...